x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -30,6 +44,34 @@ export const installTemplates = async (
   }
   return [];
 };
+
+// this is temporary until we update the registry to use index templates v2 structure
```

Comment:
currently the index template are broken up into files: https://github.com/elastic/package-registry/tree/master/dev/package-examples/base-1.0.0/elasticsearch/index-template .  Until we have it in the index templates v2 structure, they are being parsed and merged.

Reply:
Why do you need to merge them? Just installing both should also work as expected.

Reply:
Right, inheritance.  I hadn't thought of it at the time. Though in this case since there is nothing to override, is there any point in making two requests?